test/suite/test_cursor_bound17.py

```diff
@@ -130,4 +130,4 @@ def test_bound_checkpoint_or_rollback(self):
 
 
 if __name__ == '__main__':
-    wttest.run()
```

Comment:
Deleted newline at end of this file.

Reply:
[23a8382](https://github.com/wiredtiger/wiredtiger/pull/8265/commits/23a8382b74954e1cbb8d5a7067b5eeb249828ebe)